tests/unit/test_data.py

```diff
@@ -147,6 +150,16 @@ def test_len(self, data_type):
         _data = data_type()
         assert len(_data) == len(_data.data)
 
+    def dtype(self, data_type):
+        """Test dtype property; required for jaxtyping annotations."""
+        _data = data_type()
+        assert _data.data.dtype == _data.dtype
+
+    def shape(self, data_type):
+        """Test shape property; required for jaxtyping annotations."""
+        _data = data_type()
+        assert _data.data.shape == _data.shape
```

Comment:
If you want these to be run as tests, you need to ensure their names start with `test_` else Pytest won't run them

Reply:
Don't know how I missed this, thanks for spotting!